qchem/setup.py

```diff
@@ -16,7 +16,14 @@
 with open("pennylane_qchem/_version.py") as f:
     version = f.readlines()[-1].split()[-1].strip("\"'")
 
-requirements = ["pennylane>=0.11", "openfermion", "openfermionpyscf", "openfermionpsi4", "pyscf<=1.7.1"]
+requirements = [
+    "pennylane>=0.11",
+    "scipy<1.5",
```

Comment:
Got it! :) Would we then want to allow equality too?
```suggestion
    "scipy<=1.5",
```

Reply:
Looks like it doesn't support scipy>=1.5! So we'll have to leave it as is